### PR TITLE
Bump to the latest tc artifacts

### PR DESCRIPTION
**Pull request by myronkscott**

### CODE REVIEW

**Comment by saurabhagas** [comment]
I need to get a few changes from tc-platform and perform a release. Can this PR wait for ~30 minutes? Want to avoid another release train..

File: gradle.properties [hunk 1]
```diff
@@ -10,10 +10,10 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.8.0-pre1
-terracottaApisVersion = 1.7.0-pre1
-terracottaCoreVersion = 5.7.0-pre25
-terracottaPassthroughTestingVersion = 1.7.0-pre12
+terracottaPlatformVersion = 5.8.0-pre4
```
Comment (mathieucarbou) [inline gradle.properties:13]: @jhouserizer @chrisdennis @myronkscott @saurabhagas @akomakom : please help ⚠️ 
I am unable to release tc-platform `5.8.0-pre4`: the releaser job is failing.

**Comment by mathieucarbou** [comment]
@chrisdennis : versions are right now, but could  you please add a commit to switch to v0.0.3 for utilities. Build is failing because of the deprecation warnings now. Thanks!

**Comment by chrisdennis** [comment]
Replaced by #2775

**Pull request closed without merging**